compiler/damlc/daml-ide-core/test/Development/IDE/State/API/Testing.hs

```diff
@@ -126,15 +126,15 @@ data ExpectedChoiceAction
     = Create TemplateName
     | Exercise TemplateName ChoiceName deriving (Eq, Ord, Show)
 
-data ExpectedChoices = ExpectedChoices
+data ExpectedChoice = ExpectedChoice
     { _cName :: String
     , _consuming :: Bool
+    , _action :: Set.Set ExpectedChoiceAction
```

Comment:
You might want to add an unqualified import `import Data.Set (Set)` so you can avoid having to write `Set.Set` everywhere.